data/json/mapgen/nuclear_plant/nuclear_plant_z0.json

```
@@ -0,0 +1,371 @@
+        "F................................................................................................................................./I        I  I QQIVV  I/...................H.........................................H.................../IVVVVIQQQI  I        I/...............................//aaaaaaaAAaaaaaaa//.........................................................................F",
+        "F................................................................................................................................./I oo  oo I  +   +   zI/...................H.........................................H.................../I    +   +  I oo  oo I/...............................//aaaaaaaAAaaaaaaa//.........................................................................F",
+        "F...1..............................................................................................................////..//////////I        I  IIIIIIIIII/...................H.........................................H.................../IIIIIIIIII  I        I//////////...////...............//aaaaaaaAAaaaaaaa//.....................................................................1...F",
+        "F................................................................................................................../ff/../IIIIIIIIIIIIIIIII+I  IwwwI27I7I/...................H.........................................H.................../I7I72IwwwI  I+IIIIIIIIIIIIIIIII/.../ff/...............//aaaaaaaaaaaaaaaa//.........................................................................F",
+        "F................................................................................................................../f</../I                    +   + %  I/...................H.........................................H.................../I    +  %+                    I/.../<f/...............//aaaaaaaAAaaaaaaa//.........................................................................F",
+        "F...1...........................................................................................................////f/////I                    Iu  I27I7I////////////////////H/////////////////////////////////////////H////////////////////I7I72I  uI                    I//////f///.............//aaaaaaaAAaaaaaaa//.....................................................................1...F",
+        "F.............................................................................................................../IIII$IIIIII+++IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII$$$$$IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII$$H+$IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII+++IIIIII$IIII/.............//aaaaaaaAAaaaaaaa//.........................................................................F",
```

```suggestion
        "F.............................................................................................................../IIII$IIIIII+++IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII$$$$$IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII$$H+$IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII+++IIIIII$IIII/.............//aaaaaaaAAaaaaaaa//.........................................................................F",
```
Characters 120 and 273 here (I believe, unless I've gotten lost in this impressive mapgen), the `$` `t_door_metal_locked`, appear to be the only two doors made for people who work at the plant to enter. There's one on either side. There are other openings for train tracks, but it just struck me as I was playing that there were literally _zero_ emergency exits and the ones that are there are _locked_!

I would suggest adding an electric card reader that takes Industrial ID cards next to these doors, or perhaps a chance for them to be open. It might also be worth adding extra doors, but I realize you probably designed routes through this place intentionally.

---

This wall, that you mentioned, it's outer wall of the power unit. This wall may be the so-called dirty wall. Behind it are some radioactive rooms, so-called dirty rooms. So it is not intended for personnel entry. But I'm not a power plant worker, so I could be wrong.

Cleaner (no radioactive) rooms are located on the north side of the power plant. This is a deaerator shelf (longest building in the middle) and also this is a engine room. So personnel should enter through the side of the deaerator and go straight up to the second floor, I guess.

Also, in reality, somewhere there should be a pedestrian gallery to the administrative building, which is not presents now.

Adding an industrial card to some doors would be a good idea, but I'm not sure this card exists now. This pull request is already too large. Maybe we'll add it later?

---

Sure! You're welcome to ignore it if you like; I just wanted to note it and suggest one possibility.

The card reader you'd be looking for is called `t_card_industrial`. The Irradiation Plant and the Prison already use them; not sure if anything else. But the Industrial ID (`id_industrial`) cards that they use drop fairly consistently in the wallets of zombie technicians, so that was my thought! Kill a few zombies, escape out the back.

---

Thanks, I'll see what I can do.

---

So, I'd added some industrial readers to map. In my opinion, it has much easier to get out and get inside in a power plant. So I added some more closed doors. But the balance is somewhat broken anyway if you have a lot of id cards.